### ISSUE TITLE
v2 doesn't work with CRA's browserslist setup

### ISSUE THREAD

**Issue by stramel** [doczjs/docz#966]
## Bug Report

**Describe the bug**

When using the v2 of Docz with CRA v3's `browserslist`.

CRA's `browserslist` setup is for `development` and `production`.

```json
{
	"browserslist": {
    	"production": [
      		">0.2%",
      		"not dead",
      		"not ie <= 11",
      		"not op_mini all"
    	],
    	"development": [
      		"last 1 chrome version",
      		"last 1 firefox version",
      		"last 1 safari version"
    	]
  	}
}
```

But Docz v2 is expecting something like:

```json
{
	"browserslist": [
      	"last 1 chrome version",
      	"last 1 firefox version",
      	"last 1 safari version"
    ]
}
```

**Expected behavior**

I expect to be able to use Docz v2 with my current CRA `browserslist` setup

**Comment by afshin-hoseini** [comment]
I have same error. What's the work around?

**Comment by easychen** [comment]
same +1

**Comment by rakannimer** [comment]
Hey,

This issue was fixed in version 2.0.0-rc.33. 

Any version after that should work as you'd expect it.

Feel free to re-open if it happens again
